_posts/2020/2021-01-07-how-to-animate-a-svg-with-css.md

```diff
@@ -0,0 +1,149 @@
+
+The second property is _transform-origin_. With this, we change the default origin and set another anchor point for an element's transformation.
+
+As you can see in our example, the wheels now behave as desired: [SVG rotate origin edited](https://codepen.io/zefia/pen/LYRjBKe){:target="_blank"}.
+
+<iframe height="400" style="width: 100%;" scrolling="no" title="SVG rotate origin changed" src="https://codepen.io/zefia/embed/LYRjBKe?height=400&theme-id=dark&default-tab=css,result" frameborder="no" loading="lazy" allowtransparency="true" allowfullscreen="true">
+  See the Pen <a href='https://codepen.io/zefia/pen/LYRjBKe'>SVG rotate origin changed</a> by Svenja Völz
+  (<a href='https://codepen.io/zefia'>@zefia</a>) on <a href='https://codepen.io'>CodePen</a>.
+</iframe>
+
+Please keep in mind that this is just an introduction to get started with SVG animations.
+So, forgive me if we don't go into too much detail or if some things are broken down.
+
+## How to animate SVG strokes
+
+Similar to the dashed value for borders in CSS, you can use the _stroke-dasharray_ property for SVGs to create dashes in the stroke. A higher number increases the space between the strokes.
+
+The _stroke-dashoffset_ property defines along an SVG path where the stroke of a dashed line starts. Here, a higher number means that the strokes will start further along the path.
+
+With these two properties and a simple animation, the magic begins!
+A dash can be as long as we want it to be and if it is longer than the path length, it looks like our shape is not dashed at all.
+If we now set an offset that is high enough, it looks like there is no shape.
+Now, we just need to set this offset back to 0 in an animation.
+And ta-da!
+It looks like the graphic is drawing itself: [SVG writing animation](https://codepen.io/zefia/pen/RwGZJYK){:target="_blank"}.
+
+<iframe height="400" style="width: 100%;" scrolling="no" title="SVG writing animation" src="https://codepen.io/zefia/embed/RwGZJYK?height=400&theme-id=dark&default-tab=html,result" frameborder="no" loading="lazy" allowtransparency="true" allowfullscreen="true">
+  See the Pen <a href='https://codepen.io/zefia/pen/RwGZJYK'>SVG writing animation</a> by Svenja Völz
+  (<a href='https://codepen.io/zefia'>@zefia</a>) on <a href='https://codepen.io'>CodePen</a>.
+</iframe>
+
+Now you know the basics to easily and quickly animate an SVG with CSS.
+If you want to dive further and implement more complex animations, you can use JavaScript to enhance your animation or have a look at these JavaScript libraries that offer even more options:
+
+- [GreenSock](https://greensock.com/){:target="_blank"}
+- [Snap.svg](http://snapsvg.io/){:target="_blank"}
+- [anime.js](http://animejs.com/){:target="_blank"}
```

Comment:
In a sentence, they also say "Anime.js". Thus, I'd go with this to be consistent